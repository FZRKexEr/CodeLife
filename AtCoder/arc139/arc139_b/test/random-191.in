100
81 12 8 4 18 7
63 1 4 18 16 6
36 10 2 14 13 15
34 11 7 10 4 5
68 20 3 10 19 5
30 4 10 2 6 4
1 8 14 16 17 13
9 10 16 3 12 20
27 14 20 18 1 10
4 14 4 11 16 5
17 13 3 20 19 20
35 1 15 17 6 5
13 2 9 16 4 12
85 15 7 11 4 16
75 3 20 18 4 10
83 2 4 20 18 8
71 15 17 1 9 5
37 20 16 14 2 4
76 11 9 4 18 18
49 12 19 18 3 3
15 1 9 11 14 6
2 20 12 6 7 4
54 14 15 15 3 1
63 13 20 14 15 14
98 11 16 3 14 19
85 10 8 6 7 16
64 20 15 4 1 1
15 15 6 4 14 12
26 9 11 17 16 18
64 13 15 1 4 20
8 17 5 15 17 13
51 15 2 12 15 5
69 19 6 16 5 11
56 10 6 7 9 19
42 3 13 7 7 14
2 17 6 1 9 13
70 17 8 3 2 18
76 6 17 2 20 20
49 11 1 17 9 10
14 16 11 12 19 1
6 7 8 18 17 12
75 1 12 8 12 5
17 6 1 10 5 20
29 20 13 7 11 6
15 4 4 20 11 8
62 13 10 16 4 3
65 11 3 15 20 20
71 6 1 9 20 7
9 6 5 19 8 3
46 20 6 4 6 4
39 13 8 4 15 13
65 20 5 10 11 17
8 15 16 3 14 5
22 20 17 5 15 13
66 18 14 13 13 7
16 19 9 4 3 11
83 7 13 19 12 1
100 4 2 5 7 8
7 12 13 5 3 10
14 11 1 7 7 4
98 9 18 16 16 15
42 17 10 15 11 11
21 1 12 6 6 2
56 2 5 13 7 4
21 14 4 10 10 17
4 3 10 1 4 8
72 18 12 20 20 9
58 18 18 12 2 1
28 6 6 19 6 6
83 16 18 2 3 17
29 19 5 19 5 1
60 12 16 8 19 14
34 19 5 5 15 14
98 15 20 9 12 11
91 6 9 1 17 16
4 20 18 16 17 3
39 4 11 13 14 19
30 7 1 10 12 8
49 1 19 19 17 7
66 8 9 13 15 11
68 6 9 4 4 20
77 2 11 17 1 2
22 6 15 18 9 4
71 3 16 14 1 6
10 10 4 20 13 10
43 20 7 14 4 11
84 7 11 2 16 18
29 15 9 19 3 9
40 4 5 3 4 17
10 9 17 3 18 2
100 5 9 5 17 20
17 17 14 14 14 2
78 1 6 20 9 1
14 14 19 14 4 15
25 9 2 3 1 16
58 2 18 14 3 19
30 5 7 15 20 2
31 17 3 20 1 12
29 14 2 19 20 8
4 14 14 7 20 7
