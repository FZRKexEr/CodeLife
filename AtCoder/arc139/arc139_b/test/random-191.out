74
138
84
22
87
12
16
27
136
5
59
40
20
69
50
198
23
20
130
30
72
12
138
84
120
86
19
14
104
28
58
60
37
64
49
2
14
96
86
37
108
54
60
33
84
19
175
255
65
16
45
50
24
25
66
39
101
175
35
19
203
74
56
57
57
4
54
51
100
21
7
61
57
83
91
64
97
64
201
112
52
14
31
25
13
50
140
65
40
21
225
14
13
4
23
63
38
89
59
28
